docs/gettingstarted/mint_tokens.md

```diff
@@ -16,6 +16,118 @@ nav_order: 9
 
 ---
 
-## Work in progress
+## Quick reference
 
-... the tokens APIs are under development
\ No newline at end of file
+- FireFly provides an abstraction layer for multiple types of tokens
+- Tokens are grouped into _pools_, which each represent a particular type or class of token
```

Comment:
Wonder if an example worthwhile here, such as?

Examples of how a pool might be used by a given token connector implementation:
    - ERC-721: A contract instance, ready for issuance of NFTs within unique identities in the pool
    - ERC-20 / ERC-777: A token contract instance, with a single fungible pool of value - "a coin"
    - ERC-1155: An efficient to allocate isolated pool of fungible or non-fungible tokens (see [ref implementation](https://github.com/hyperledger/firefly-tokens-erc1155))
    - ERC-1400 (and related security token standards): An partition within an overall fungible pool, representing part of the value of an overall non-fungible asset

Could do with @jimthematrix checking on the terminology above

Reply:
Could be, and maybe not limited to pools... Maybe we need an extra section of "Example uses" to lay out a few examples of how you could map common token standards/use cases onto FireFly.

Reply:
if we agree with the above, we should also add that the current API endpoints support the ERC20, ERC777, ERC1155 and ERC721 contracts, while future work is needed to support ERC1400, in particular to support partitions (metadata to fungible tokens) within pools.

Reply:
Added a new section with some of these examples.